src/main/kotlin/com/devfloor/untitled/hashtag/application/HashtagService.kt

```diff
@@ -0,0 +1,26 @@
+package com.devfloor.untitled.hashtag.application
+
+import com.devfloor.untitled.hashtag.domain.Hashtag
+import com.devfloor.untitled.hashtag.domain.HashtagRepository
+import com.devfloor.untitled.option.domain.Option
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class HashtagService (
+    private val repository: HashtagRepository
+){
+    @Transactional(readOnly = true)
+    fun show(hashtags: List <String>): List<Hashtag> {
```

Comment:
메서드가 List를 반환하는 것으로 보입니다 !
따라서 showAllByNames와 같이 메서드명을 변경하는게 어떨까요?
```suggestion
    fun showAllByNames(hashtags: List <String>): List<Hashtag> {
```